src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -559,16 +560,18 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                 // construct the Account name
                 a.Name = CAO_Constants.isHHAccountModel() ? strNameHHAccountForContact(c) : strNameOne2OneAccountForContact(c);
                 
-                //connect the Account to the Contact for 1:1 and HHAccount
+                // connect the Account to the Contact for 1:1 and HHAccount
                 if (c.Id != null) {
                     a.npe01__One2OneContact__c = c.Id;
                 }
                 a.Phone = c.Phone;
                 a.Fax = c.Fax;
+                a.OwnerId = c.OwnerId;
```

Comment:
@force2b do we need to be concerned with the owner being active here as well?

Reply:
@jjbennett  Good question. No because normally it would not be possible to insert a Contact with an Inactive Owner. However if for some reason a setting is enabled to allow an Contact insert with an inactive owner, then we'd want the Account owner to be the same.

Reply:
Ok, great thanks!

Reply:
It looks like a test is failing regarding an inactive owner.

Reply:
Working on that still. Almost there.